sdk/src/contexts/abstracts/relayer.ts

```diff
@@ -2,7 +2,10 @@ import { BigNumber, BigNumberish } from 'ethers';
 import { TokenId, ChainName, ChainId } from '../../types';
 import { TokenBridgeAbstract } from './tokenBridge';
 
-export abstract class RelayerAbstract extends TokenBridgeAbstract {
+export abstract class RelayerAbstract<
+  SendResult,
+  RedeemResult,
```

Comment:
Can we just have 1 type `TransactionResult`? I think it's the same type for both